Frameworks/Foundation/NSOperation.mm

```diff
@@ -277,35 +244,35 @@ - (void)main {
  @Status Interoperable
 */
 - (void)waitUntilFinished {
-    pthread_mutex_lock(&priv->finishLock);
+    pthread_mutex_lock(&_finishLock);
     while (![self isFinished]) {
-        pthread_cond_wait(&priv->finishCondition, &priv->finishLock);
+        pthread_cond_wait(&_finishCondition, &_finishLock);
     }
-    pthread_mutex_unlock(&priv->finishLock);
+
+    pthread_mutex_unlock(&_finishLock);
 }
 
 /**
  @Status Interoperable
 */
-- (id)dependencies {
-    return priv->dependencies;
+- (NSArray*)dependencies {
```

Comment:
It looks like the reference platform dispatches full change notifications and not indexed ones.

---
In reply to: [69615177](https://github.com/Microsoft/WinObjC/pull/602#discussion_r69615177) [](ancestors = 69615177,69358159,69357321)